src/main/kotlin/org/stellar/walletsdk/utils/validateTransaction.kt

```diff
@@ -0,0 +1,24 @@
+package org.stellar.walletsdk.utils
+
+import org.stellar.sdk.Server
+import org.stellar.sdk.Transaction
+
+fun validateTransaction(transaction: Transaction, server: Server) {
```

Comment:
This name is too generic. I don't know what it does by reading the name, can we rename it to be more explicit?

Reply:
So the idea was that this helper could be used to validate the transaction before submitting it. Then we could validate signatures, etc. We could also check it before signing. `preSubmitValidation()`? I'm open to suggestions. :pray:

Reply:
Hmm.. I don't know if we should have this function, because there are a lot of checks we could make, and these checks could be expensive.

See the errors defined in this section of the [docs](https://developers.stellar.org/api/errors/http-status-codes/horizon-specific/). I'd be concerned we're giving the developer a false sense of security because we're only checking subset of checks we could perform. Thoughts?

Reply:
Yeah, I'm on the fence about it. :thinking: I thought it might be useful to check if the account has enough balance to cover the fees, but maybe that's not necessary for a custodial wallet? I'm OK removing this helper.

Reply:
I'll keep this for now, but we can always remove it later.